### DISCUSSION TITLE
Email Notification Template - Subject - Add component and project names

### DISCUSSION THREAD

**Discussion by JN-CSIRT** [DependencyTrack/dependency-track#1811] [Q&A]
For our ticketing it is necessary to get Component and Project name into notification email's subject.
Is it possible to add them in the Template or somewhere else?

**Comment by syalioune** [comment]
Hello @JN-CSIRT,
The notification email's subject are hard coded right now. You should fill in an enhancement request for this feature [here](https://github.com/DependencyTrack/dependency-track/issues).

**Reply by JN-CSIRT** [reply]
Enhancement is created: https://github.com/DependencyTrack/dependency-track/issues/1823

**Reply by JN-CSIRT** [reply]
> Hello @JN-CSIRT, The notification email's subject are hard coded right now. You should fill in an enhancement request for this feature [here](https://github.com/DependencyTrack/dependency-track/issues).

Hi syalioune,

Is there any changes? We getting now the New CVE Email Alerts with Project names/Versions in the subject only.
How we can add component name/version into subject 
we use DT 4.7.1

**Comment by msymons** [comment]
@JN-CSIRT,  you logged this issue back in July 2022 (when v4.5.0 was the latest version) and the reply "The notification email's subject are hard coded right now" was valid at the time. 

However, v4.6.0 (released October 11 2022) introduced  the ability to customise your notification templates...

> Customization. Users with advanced customization needs can now create and modify notification templates, as well as specify custom intervals for recurring tasks. Refer to the [notifications](https://docs.dependencytrack.org/integrations/notifications/) and [recurring tasks](https://docs.dependencytrack.org/getting-started/recurring-tasks/) documentation for details.

Have you tried this functionality?   It it does what you need then this question (and linked enhancement) can be resolved.  If not, then let us know exactly what went wrong (or was just lacking) in a new issue.

**Reply by JN-CSIRT** [reply]
> Have you tried this functionality? It it does what you need then this question (and linked enhancement) can be resolved. If not, then let us know exactly what went wrong (or was just lacking) in a new issue.

Hi MSymons,
we tried it with template and Email alert content - works.
But there impossible to configure Emails Subject. 
We send Email alerts into our Ticketing system and there we sort the Tickets by Email Subject 
So it is important for us to configure the Alert Email Subject.

We did not found it in the Template configuration.
Can you point me please where is in in the Template for New CVE Found?

Thanks in advance,

**Reply by JN-CSIRT** [reply]
If you mean "notification.subject" option,
can you please help with this configuration as follow:
Subject: Poject : Project Version : Component : component Version

Currently in the Subject and in the Email content first line we have - {{ notification.title }}

**Reply by syalioune** [reply]
Hello @JN-CSIRT,
My previous answer still apply. While v4.6.0 introduced customization features, it was for the notification body content.
The notification title, thus email subject, is unfortunately still hardcoded.
Enhancement request #1823 is still valid.
